apps/src/code-studio/components/progress/CourseProgress.jsx

```diff
@@ -97,7 +97,7 @@ const CourseProgress = React.createClass({
                   <CourseProgressRow
                     stage={stage}
                     key={stage.name}
-                    isFocusArea={focusAreaPositions.indexOf(count++) > -1}
+                    isFocusArea={focusAreaStageIds.includes(stage.id)}
```

Comment:
I'm inclined to leave this as it is; feels too much like a premature optimization